src/bin/adbackup-cli.rs

```diff
@@ -102,8 +104,35 @@ fn make_clap<'a, 'b>() -> clap::App<'a, 'b> {
                 .help("List all android devices connected to your pc with enabled debug mode."),
         )
         .subcommand(
-            SubCommand::with_name("apps")
+            SubCommand::with_name("pull")
                 .display_order(3)
+                .about("Pull file/folder from device")
+                .arg(device_arg())
+                .arg(
+                    Arg::with_name("target")
+                        .help("Target file/folder")
+                        .required(true),
+                )
+        )
+        .subcommand(
+            SubCommand::with_name("push")
+                .display_order(4)
+                .about("Push file/folder to device")
+                .arg(device_arg())
+                .arg(
+                    Arg::with_name("source")
```

Comment:
maybe clarify which of those two (`source` / `target`) is on host machine (unix / max / windows) and which is on the android device